src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -190,10 +200,34 @@ class EthService(
   }
 
   def submitHashRate(req: SubmitHashRateRequest): ServiceResponse[SubmitHashRateResponse] = {
-    //todo do we care about hash rate for now?
+    hashRate.updateAndGet(new UnaryOperator[Map[ByteString, (BigInt, Date)]] {
+      override def apply(t: Map[ByteString, (BigInt, Date)]): Map[ByteString, (BigInt, Date)] = {
+        val now = new Date
+        clearTimeouts(now, t + (req.id -> (req.hashRate, now)))
```

Comment:
Should we check here or in `EthJsonMethodImplicits` that `req.id` and `req.hashrate` have a length of 32 bytes as mentioned in https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_submithashrate